### PR TITLE
Feat/color scale legend

### PR DESCRIPTION
**Pull request by Bogdan2423**
looks like this:
Przechwytywanie

### CODE REVIEW

**Review by kacperklusek — APPROVED** [review]

File: src/components/shared/Map/Legend.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  }> = [];
+
+  for (let i = 0; i < pollutionLevels.length; i += 1) {
+    const levelData = {
+      id: i,
+      color: COLORS.pollutionScale[i],
+      description: pollutionLevels[i]
+    };
+    legendData.push(levelData);
+  }
```
Comment (MatiPl01) [inline src/components/shared/Map/Legend.tsx:26]:
```suggestion
const legendData = pollutionLevels.map((level, i) => ({
    id: i,
    color: COLORS.pollutionScale[i],
    description: level
  }));
```

Cleanup code

File: src/components/shared/Map/Legend.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+  return (
+    <div className='absolute top-40 right-4 bg-black bg-opacity-50 rounded z-10'>
```
Comment (MatiPl01) [inline src/components/shared/Map/Legend.tsx:29]:
```suggestion
    <div className='absolute top-4 right-4 bg-black bg-opacity-50 rounded z-10'>
```
Maybe make distances the same (from top and right screen side). It looks a bit strange when a distance from the top is so large.

**Review by MatiPl01 — DISMISSED** [review]
Looks good. Suggested small changes.

File: src/components/shared/Map/Legend.tsx (hunk 2 above)
Comment (Bogdan2423) [reply src/components/shared/Map/Legend.tsx]: alright

**Review by MatiPl01 — APPROVED** [review]
Looks good

**Review by kosmydel — APPROVED** [review]

**Pull request merged**